rust/src/web/mod.rs

```diff
@@ -31,6 +37,16 @@ pub async fn start_web_server<A: net::ToSocketAddrs, P: AsRef<Path>>(
             .service(blocks::get_block)
             .service(accounts::get_account)
             .service(blockchain::get_blockchain_summary)
+            .service(
+                web::resource("/graphql")
+                    .guard(guard::Post())
+                    .to(GraphQL::new(schema.clone())),
+            )
+            .service(
+                web::resource("/graphql")
```

Comment:
Should we make "/graphql" a `const`?

Example: `pub const ENDPOINT_GRAPHQL: &str = "/graphql";`